### PR TITLE
fix: mobile suggestion wrap

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Fix the wrapping of recommendation.

Preview:
image
Screenshot 2023-08-29 at 6 31 26 PM

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1708 #done

### CODE REVIEW

**Review by rebelchris — CHANGES_REQUESTED** [review]
I think this task was mainly about missing the top buttons on the actual device.
Does this solve that issue as well?

**Comment by sshanzel** [comment]
> I think this task was mainly about missing the top buttons on the actual device.
Does this solve that issue as well?

Oh, there was no mention of what to fix and that overflow was the one that popped out. Let me check that as well.

File: packages/shared/src/components/tooltips/InteractivePopup.tsx [hunk 1]
```diff
@@ -37,6 +38,7 @@ const positionClass: Record<InteractivePopupPosition, string> = {
   leftStart: classNames(leftClass, startClass),
   leftCenter: classNames(leftClass, centerClassY),
   leftEnd: classNames(leftClass, endClass),
+  screen: 'inset-0 w-screen h-screen',
```
Comment (sshanzel) [inline packages/shared/src/components/tooltips/InteractivePopup.tsx:41]: At line 32, there are certain classes for `center` position that we don't need. So we introduce a specific one for the case we have now.

**Comment by sshanzel** [comment]
@rebelchris have updated it now 🥳

File: packages/shared/src/components/tooltips/InteractivePopup.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/tooltips/InteractivePopup.tsx:41]: Ah right, something modal ofcourse does, good to know for future I guess :)

**Review by rebelchris — APPROVED** [review]

**Pull request merged**